cmd/config/create_test.go

```diff
@@ -0,0 +1,71 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/screwdriver-cd/sd-local/config"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConfigCreateCmd(t *testing.T) {
+	rand.Seed(time.Now().UnixNano())
+	cnfPath := fmt.Sprintf("%vconfig", rand.Int())
+	defer os.Remove(cnfPath)
+
+	cnew := configNew
+	defer func() {
+		configNew = cnew
+	}()
+	configNew = func(configPath string) (c config.Config, err error) {
+		return config.New(cnfPath)
+	}
+
+	testCase := []struct {
```

Comment:
How about to add test case which try to create entry that was already created.

Reply:
I added that test case.